lease/lessor.go

```diff
@@ -381,21 +382,19 @@ func (le *lessor) initAndRecover() {
 			expiry: forever,
 		}
 	}
-	tx.Unlock()
 
-	le.b.ForceCommit()
-}
+	// get last granted lease (if any) to seed grant values
+	_, vs = tx.UnsafeRange(leaseBucketName, lastLeaseKey, lastLeaseKey, 0)
+	if len(vs) != 0 {
```

Comment:
`Just question. Would this always return 1 value `if len(vs) != 0` because we keep overwriting `lastLeaseKey`?